### PR TITLE
Internal Imagenet normalisation for pretrained Vgg

### PR DESCRIPTION
**Pull request by ekagra-ranjan**
Makes it easier to normalise the model weights with imagenet mean and std when using transfer learning. Useful for beginners who forget to normalise the input images while using transfer learning. Consistent with googleNet and inceptionV3 pytorch implementation.

### CODE REVIEW

**Comment by fmassa** [comment]
Same comment from https://github.com/pytorch/vision/pull/782#issuecomment-471199627 applies
